project-base/storefront/components/Pages/App/CartReloader.tsx

```diff
@@ -0,0 +1,7 @@
+import { useReloadCart } from 'hooks/cart/useReloadCart';
+
+export const CartReloader: FC = () => {
+    useReloadCart();
+
+    return null;
+};
```

Comment:
Is it necessary to have component for this? I don't see much of the point to have those "microcomponents" with one hook. Why not use it directly in AppPageContent?